src/geometry/manifold/mfderiv.lean

```diff
@@ -696,6 +696,12 @@ end
 
 omit Is I's
 
+lemma mdifferentiable_at.prod_mk {f : M → M'} {g : M → M''} {x : M}
```

Comment:
Do we have this for `mdifferentiable_within_at`, `mdifferentiable_on`, and `mdifferentiable`?
BTW, there is a question to think about: if the codomains are normed spaces, then we have two models on their product. Should we have API for both `prod`? E.g., for `cont_mdiff` we have `prod_mk` and `prod_mk_space` lemmas.